src/opossum_lib/cli.py

```diff
@@ -63,6 +65,10 @@ def spdx2opossum(infile: str, outfile: str) -> None:
 
     if not outfile.endswith(".opossum"):
         outfile += ".opossum"
+
+    if Path.is_file(Path(outfile)):
+        logging.warning(f"{outfile} already exists and will be overwritten.")
```

Comment:
Is this really the behaviour that we want here? We could also require a non-existing file and else throw an error.
Depends how this is used, I guess. If the script is run multiple times with the default output file, it would be overwritten and only the last one would survive. Is this what we want?